selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/organization/OrganizationPage.java

```diff
@@ -54,7 +54,7 @@
     String MEMBERS_TAB = "//md-tab-item//span[contains(text(), 'Members')]";
     String MEMBERS_LIST = "//list-organization-members";
     String ADD_MEMBER_BUTTON = "//che-button-primary[@ng-click = 'onAdd()']";
-    String DELETE_MEMBER_BUTTON = "//button[@ng-click = 'onDelete()']";
+    String DELETE_MEMBER_BUTTON = "delete-item-button";
```

Comment:
It's button's id. So, it will be more obviouse if we rename:
`DELETE_MEMBER_BUTTON` > `DELETE_MEMBER_BUTTON_ID`